<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestProcWdg</class>
 <widget class="QWidget" name="TestProcWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <widget class="QLabel" name="lbAverageScore">
      <property name="geometry">
       <rect>
        <x>538</x>
        <y>23</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgba(80,80,0,30);
font-size: 12px; 
font-weight: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>23</y>
        <width>82</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Average score</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>42</y>
        <width>82</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Group level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbGroupLevel">
      <property name="geometry">
       <rect>
        <x>538</x>
        <y>42</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgba(80,80,0,30);
font-size: 12px; 
font-weight: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>4</y>
        <width>90</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>High level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbHighPercent">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>4</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgba(80,80,0,30);
font-size: 12px; 
font-weight: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbMiddlePercent">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>23</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgba(80,80,0,30);
font-size: 12px; 
font-weight: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>23</y>
        <width>90</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Middle level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>112</x>
        <y>42</y>
        <width>90</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Low level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbLowPercent">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>42</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgba(80,80,0,30);
font-size: 12px; 
font-weight: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>4</y>
        <width>112</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum points</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbMaxPoints">
      <property name="geometry">
       <rect>
        <x>538</x>
        <y>4</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>The number of points that can be scored by answering correctly all questions of the questionnaire</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background : rgba(80,80,0,30);
font-size: 12px; 
font-weight: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
